packages/viz/src/lib/style/helpers/color-bins-style.ts

```diff
@@ -0,0 +1,35 @@
+import { getColors, validateParameters } from './utils';
+
+export function colorBinsStyle(
+  featureName: string,
+  { bins = defaultOptions.bins, binColors = defaultOptions.binColors }: ColorBinStyleOptions = defaultOptions
```

Comment:
Nit, should we change ColorBinStyleOptions to ColorBinsStyleOptions with 's'?

Reply:
Yes, sure! We need to be consistent!